### DISCUSSION TITLE
Videos that belong to multiple playlists in a collection

### DISCUSSION THREAD

**Discussion by brianroberg** [jmbannon/ytdl-sub#826] [Q&A] [answered]
I'm using a TV Show Collection to download a channel and present each of its playlists in Jellyfin as a season. This is working great! The playlists-as-seasons under one show works very well as a concept. But I'm wondering if it's possible to tweak what happens in the case of a video that appears in more than one playlist. 

Here's my use case: my wife uses a fitness channel that has 20+ playlists. The playlists serve as categories of workout types, but the categories are far from mutually exclusive—many videos appear in more than one list. What I want is for a video that appears in more than one playlist on YouTube to be accessible in *all* those playlists in Jellyfin, thus preserving the organization scheme the YouTuber devised for the channel. The behavior I'm actually getting—in keeping with what's described in the documentation—is that a video appears in only one playlist.

Of course, I don't want to toss out the `maintain_download_archive` features altogether; I don't want to download and store redundant copies of all those videos. It seems like an ideal solution would be to create symlinks or hard links to the first copy of a video when it appears in additional playlists. Is this possible? Or is there perhaps a different way to accomplish the same result?

I really appreciate any help anyone can offer. I have almost everything working well, but this last piece would be big for the Spouse Approval Factor!

**Comment by jmbannon** [comment, marked as answer]
Hey @brianroberg , follow this issue for a solution to this problem (hardlink/symlink support for dupe videos is a requirement for this): https://github.com/jmbannon/ytdl-sub/issues/502

It's something I've thought about and want to implement at some point. Glad you are enjoying ytdl-sub!

**Reply by brianroberg** [reply]
OK, thank you! I came across that issue while I was setting up the collection, but I hadn't discerned the connection to hardlinks/symlinks.

Thanks for all the work you're doing on this project!
